internal/cmdrender/executor.go

```diff
@@ -193,14 +211,19 @@ type pkgNode struct {
 }
 
 // newPkgNode returns a pkgNode instance given a path or pkg.
-func newPkgNode(path string, p *pkg.Pkg) (pn *pkgNode, err error) {
+func newPkgNode(path string, p *pkg.Pkg, fs filesys.FileSystem) (pn *pkgNode, err error) {
 	const op errors.Op = "pkg.read"
 
 	if path == "" && p == nil {
 		return pn, fmt.Errorf("missing package path %s or package", path)
 	}
 	if path != "" {
-		p, err = pkg.New(path)
+		if fs == nil {
```

Comment:
why will fs be nil ? 
I think it will be reasonable to assume `fs` to be mandatory in method such as this to truly abstract out fs. It will be very difficult to reason for these mix-mode scenario. Callers will always ensure fsys is present (or defaulted to FileSystemOrDisk). WDYT ?

Reply:
Updated.